### ISSUE TITLE
Update Italian translations

### ISSUE THREAD

**Issue by michelelauni** [squidfunk/mkdocs-material#4981]
### Translations

```Jinja
{% macro t(key) %}{{ {
  "language": "it",
  "action.edit": "Modifica",
  "action.skip": "Vai al contenuto",
  "action.view": "Vedi il sorgente di questa pagina",
  "announce.dismiss": "Non mostrare più",
  "blog.archive": "Archivia",
  "blog.categories": "Categorie",
  "blog.categories.in": "in",
  "blog.continue": "Continua a leggere",
  "blog.draft": "Bozza",
  "blog.index": "Torna all'indice",
  "blog.meta": "Metadati",
  "blog.references": "Collegamenti",
  "clipboard.copy": "Copia",
  "clipboard.copied": "Copiato",
  "consent.accept": "Accettazione",
  "consent.manage": "Gestisci le opzioni",
  "consent.reject": "Rifiuta",
  "footer": "Piede",
  "footer.next": "Prossimo",
  "footer.previous": "Precedente",
  "header": "Intestazione",
  "meta.comments": "Commenti",
  "meta.source": "Sorgente",
  "nav": "Navigazione",
  "readtime.one": "1 minuto di lettura",
  "readtime.other": "# minuti di lettura",
  "rss.created": "Feed RSS",
  "rss.updated": "Contenuto aggiornato del feed RSS",
  "search": "Cerca",
  "search.placeholder": "Cerca",
  "search.share": "Condividi",
  "search.reset": "Cancella",
  "search.result.initializer": "Inizializza la ricerca",
  "search.result.placeholder": "Scrivi per iniziare a cercare",
  "search.result.none": "Nessun documento trovato",
  "search.result.one": "1 documento trovato",
  "search.result.other": "# documenti trovati",
  "search.result.more.one": "1 altro in questa pagina",
  "search.result.more.other": "# altri in questa pagina",
  "search.result.term.missing": "Non presente",
  "select.language": "Seleziona la lingua",
  "select.version": "Seleziona la versione",
  "source": "Apri repository",
  "source.file.contributors": "Contributori",
  "source.file.date.created": "Creata",
  "source.file.date.updated": "Ultimo aggiornamento",
  "tabs": "Tabs",
  "toc": "Indice",
  "top": "Torna su"
}[key] }}{% endmacro %}
```

### Before submitting

- [X] I've checked the list of [available languages](https://squidfunk.github.io/mkdocs-material/setup/changing-the-language/#site-language) for existing translations.
- [X] I assure that the translations are accurate to my best knowledge.
- [ ] __Optional__: I want to integrate this translation myself and create a pull request following the [contribution guide](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md).

**Comment by squidfunk** [comment]
Thanks! Added in 4b34e08c4.

**Comment by squidfunk** [comment]
Released as [9.0.11](https://github.com/squidfunk/mkdocs-material/releases/tag/9.0.11).
